exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/package-info.java

```diff
@@ -117,12 +134,14 @@
  * are added while loading, their index is always at the end of the existing
  * columns.
  * <h4>Writing Data to the Batch</h4>
- * Each batch is delimited by a call to {@link #startBatch()} and a call to
- * {@link #harvestWithLookAhead()} to obtain the completed batch. Note that readers do not
+ * Each batch is delimited by a call to {@link org.apache.drill.exec.physical.rowSet.ResultSetLoader#startBatch()}
+ * and a call to {@link org.apache.drill.exec.physical.rowSet.impl.VectorState#harvestWithLookAhead()}
+ * to obtain the completed batch. Note that readers do not
  * call these methods; the scan operator does this work.
  * <p>
- * Each row is delimited by a call to {@link #startValue()} and a call to
- * {@link #saveRow()}. <tt>startRow()</tt> performs initialization necessary
+ * Each row is delimited by a call to {@link org.apache.drill.exec.vector.accessor.writer.WriterEvents#startRow()}
```

Comment:
WriterEvents is meant to be entirely internal; it should not be part of the API. WriterEvents is the internal implementation of the public API described here.